kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/helpers/ApiHelper.java

```diff
@@ -79,40 +80,38 @@ public void createSchema(String clusterName, String schemaName, SchemaType type,
         }
     }
 
-    @SneakyThrows
     public void deleteSchema(String clusterName, String schemaName) {
         try {
             schemaApi().deleteSchema(clusterName, schemaName).block();
         } catch (WebClientResponseException ignore) {
         }
     }
 
-    @SneakyThrows
     public void deleteConnector(String clusterName, String connectName, String connectorName) {
         try {
             connectorApi().deleteConnector(clusterName, connectName, connectorName).block();
         } catch (WebClientResponseException ignore) {
         }
     }
 
-    @SneakyThrows
     public void createConnector(String clusterName, String connectName, String connectorName, String configJson) {
         NewConnector connector = new NewConnector();
         connector.setName(connectorName);
-        Map<String, Object> configMap = new ObjectMapper().readValue(configJson, HashMap.class);
-        connector.setConfig(configMap);
+        Map<String, Object> configMap = null;
         try {
-            connectorApi().deleteConnector(clusterName, connectName, connectorName).block();
-        } catch (WebClientResponseException ignored) {
+            configMap = new ObjectMapper().readValue(configJson, HashMap.class);
+        } catch (JsonProcessingException e) {
+            e.printStackTrace();
         }
+        connector.setConfig(configMap);
+            connectorApi().deleteConnector(clusterName, connectName, connectorName).block();
         connectorApi().createConnector(clusterName, connectName, connector).block();
     }
 
     public String getFirstConnectName(String clusterName) {
-        return connectorApi().getConnects(clusterName).blockFirst().getName();
+        return Objects.requireNonNull(connectorApi().getConnects(clusterName).blockFirst()).getName();
```

Comment:
this method is not used, that's why we can't say that current String can't be null in case when object is null. changes in current line are our of scope, please revert them

Reply:
Updated last PR , please review it